falcon/cmd/inspect_app.py

```diff
@@ -27,7 +27,7 @@
 
 
 def make_parser():
-    """Creates the parsed or the application"""
+    """Create the parsed or the application."""
```

Comment:
Create the ~~parser~~ parser or the application.
(But it was this way before, so not required to change in this PR)